docs/install.md

```diff
@@ -1,29 +1,43 @@
----
-layout: default
-title: Installing Compose
----
+page_title: Installing Compose
+page_description: How to intall Docker Compose
+page_keywords: compose, orchestration, install, installation, docker, documentation
 
-Installing Compose
-==============
 
-First, install Docker version 1.3 or greater.
+## Installing Compose
 
-If you're on OS X, you can use the [OS X installer](https://docs.docker.com/installation/mac/) to install both Docker and boot2docker. Once boot2docker is running, set the environment variables that'll configure Docker and Compose to talk to it:
+To install Compose, you'll need to install Docker first. You'll then install
+Compose with a `curl` command. 
+
+### Install Docker
+
+First, you'll need to install Docker version 1.3 or greater.
+
+If you're on OS X, you can use the
+[OS X installer](https://docs.docker.com/installation/mac/) to install both
+Docker and the OSX helper app, boot2docker. Once boot2docker is running, set the
+environment variables that'll configure Docker and Compose to talk to it:
 
     $(boot2docker shellinit)
 
-To persist the environment variables across shell sessions, you can add that line to your `~/.bashrc` file.
+To persist the environment variables across shell sessions, add the above line
+to your `~/.bashrc` file.
+
+For complete instructions, or if you are on another platform, consult Docker's
+[installation instructions](https://docs.docker.com/installation/).
 
-There are also guides for [Ubuntu](https://docs.docker.com/installation/ubuntulinux/) and [other platforms](https://docs.docker.com/installation/) in Docker’s documentation.
+### Install Compose
 
-Next, install Compose:
+To install Compose, run the following `curl` command:
```

Comment:
Actually [pushes bridge of glasses] this is a `curl` followed by a `chmod`. Perhaps we should stop obfuscating and split them onto two lines?